shared/config.go

```diff
@@ -9,15 +9,12 @@ import (
 )
 
 var Config ParetoConfig
-var configPath string
-
-type CheckStatus struct {
-	Disabled bool
-}
+var ConfigPath string
 
 type ParetoConfig struct {
-	TeamID    string
-	AuthToken string
+	TeamID        string
+	AuthToken     string
+	DisableChecks []string
```

Comment:
Maybe `DisabledChecks` is clearer? Sounds less like a function...
```suggestion
	DisabledChecks []string
```

Reply:
My thinking was it's instruction in specification and not actual state; state would then be disabled checks.